all-dogs.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> All Dogs</title>
+ <script src="https://kit.fontawesome.com/2315930367.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+
+  <link rel="stylesheet" type= "text/css" href= "./styles/styles.css">
+  <script src="./script.js"></script>
+</head>
+<body>
+  <header>
+
+    <div class= "checkout">
+      <i class="fas fa-shopping-cart"></i>
```

Comment:
For this you can use the asset we provide. It's good practice to only bring in outside things like this if you don't already have them.